tools/gulptasks/lib/api/server.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable require-jsdoc */
+
+// Dependencies sorted by require path
+const fs = require('fs');
+const http = require('http');
+const { parse, resolve } = require('path');
+const { promisify } = require('util');
+
+/* Create promise versions of utility functions
+ * Note: Consider e.g require('fs').promises, if limiting support to NodeJS v10
+ * and above. */
+const stat = promisify(fs.stat);
+const readFile = promisify(fs.readFile);
+
+// Constants
+const { argv } = process;
+const PORT = Number(argv[(argv.indexOf('--port') + 1)]) || 8080;
+const BASE = `http://localhost:${PORT}`;
+const PRODUCT = ['highcharts', 'highstock', 'highmaps', 'gantt'].find(
+    product => {
+        try {
+            fs.statSync(resolve(__dirname, product));
+            return true;
+        } catch {
+            return false;
+        }
+    }
+);
+const MIME_TYPE = {
+    '.css': 'text/css',
+    '.html': 'text/html',
+    '.js': 'text/javascript',
+    '.json': 'application/json',
+    '.png': 'image/png',
+    './svg': 'image/svg+xml'
```

Comment:
This one doesn't seem to be working, or at least the logo `highcharts.svg` is not working properly. It seems like the server is looking for `highcharts.svg.html` on the file system.